### PR TITLE
Emadul

### PR DESCRIPTION
**Pull request by MdEmadul**
New code added. Please review and confirm

### CODE REVIEW

File: src/test/java/Runner_File/Test_Login_Runner.java [hunk 1]
```diff
@@ -12,7 +13,6 @@
 
 		glue = { "Steps_File" }, // needs to mention here only steps package name(no java file name)
 		monochrome = true, 
-
```
Comment (salim87) [inline src/test/java/Runner_File/Test_Login_Runner.java:15]: reduce the gap

File: src/test/java/Runner_File/Test_Login_Runner.java [hunk 2]
```diff
@@ -12,7 +13,6 @@
...
 		dryRun = false
 	//	tags="@Smoketest or @RegressionTest"
```
Comment (salim87) [inline src/test/java/Runner_File/Test_Login_Runner.java]: remove the unused code

File: src/test/java/Steps_File/Login_Step.java [hunk 3]
```diff
@@ -22,12 +23,13 @@ public void user_landed_on_bank_homepage() {
 		driver.manage().window().maximize();
 		driver.manage().timeouts().implicitlyWait(Duration.ofMillis(5000));
 		System.out.println("printing landing page");
-		System.out.println("new commit");
+
```
Comment (salim87) [inline src/test/java/Steps_File/Login_Step.java]: reduce the gap

**Comment by emadulhaque** [comment]
resolved

File: src/test/java/Runner_File/Test_Login_Runner.java (hunk 2 above)
Comment (MdEmadul) [reply src/test/java/Runner_File/Test_Login_Runner.java]: Removed

File: src/test/java/Steps_File/Login_Step.java (hunk 3 above)
Comment (MdEmadul) [reply src/test/java/Steps_File/Login_Step.java]: gap reduced

**Pull request merged**